asset/js/setup.js

```diff
@@ -95,6 +95,21 @@ function setupSiteNav() {
   );
 }
 
+function setupReadingProgress() {
+  const h = document.documentElement;
+  const b = document.body;
+  const st = 'scrollTop';
+  const sh = 'scrollHeight';
+  const progress = document.querySelector('.progress');
+  let scroll;
```

Comment:
Please correct me if I'm wrong, but could we make this a `const` within the event listener instead? We don't seem to need the previous scroll value to compute the new one.

Reply:
Yes, you're right. :) Resolved.